packages/web/src/components/search/SearchBox.js

```diff
@@ -1558,6 +1578,8 @@ const SearchBox = (props) => {
 	}, []);
 
 	const hasSuggestions = () => Array.isArray(parsedSuggestions()) && parsedSuggestions().length;
+	// eslint-disable-next-line react/prop-types
+	const isTestingAI = props.testMode && props.enableAI;
```

Comment:
@SavvyShah Let's avoid disabling lint rule by doing the needful

Reply:
OK